src/ui/public/vis/editors/components/editor_options_group.js

```diff
@@ -18,10 +20,10 @@ import {
  * to produce an aligned look and feel.
  */
 function EditorOptionsGroup(props) {
-  return (
+  const collapsibleOptionGroup = (
     <EuiPanel
       grow={false}
```

Comment:
I think we want to use `prop.grow` here too :-)

Reply:
grow on panel with accordion doesn't make any sense as accordion does not grow

Reply:
In that case I would add the following to the propType definition of `grow`: "Only applies when `collapsible` is set to `false`."